src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -2312,3 +2312,102 @@ plugin could be rewritten like this:
       return [MuteController]
 
 
+Style Guide for UI
+------------------
+The style guide is created to document ceph dashbaord standards and maintain
+consistency throught the project. Its an effort to make it easier for
+contributors to process designing and deciding mockups and designs for
+Dashboard.
+
+The development environment for ceph dashbaord has live reloading enabled so
+any changes made in UI are reflected in open browser windows. Ceph dashboard
+uses bootstrap as the main third-party CSS library.
+
+Avoid duplication of code. Be consistent with the existing UI by reusing
+existing scss declarations as much as possible.
+
+Colors
+~~~~~~
+All the colors used in ceph dashboard UI are listed in `frontend/src/styles`.
+If using new color always define color variables in `frontend/src/styles` and
+use the variable instead of hard coded color values so that changes to color
+are reflected in similar UI elements.
+
+The main color for the ceph dashboard is `$color-primary`. The primary color is
+used in navigation components and as the `border-color` for input components of
+form.
+
+The secondary color is `$color-secondary` and is the background color for ceph
+dashbaord.
+
+Buttons
+~~~~~~~
+
+Buttons are used for performing actions such as: “Submit”, “Edit, “Create" and
+“Update”. These actions almost always occur on the same page.
+
+**Forms:** When using to submit forms anywhere in the dashboard, the main action
```

Comment:
It would be ideal if you could add which button should be on which side. `Cancel` always left `Perform the action` always right.